src/villas-fpga-cat.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
Do we really need this script?

Reply:
Also the name _cat_ is misleading in my opinion, as its not doing what the Unix `cat` command is doing (con**cat**enating)

Reply:
this was already in the repo before and I want to avoid removing something without replacement.